docs/security.rst

```diff
@@ -310,3 +310,20 @@ standard port 443, you'll need to configure that too. Be aware that super user p
     # Optionally, set the server to listen on the standard SSL port.
     web_server_port = 443
     base_url = http://<hostname or IP>:443
+
+Impersonation
+'''''''''''''
+
+Airflow has the ability to impersonate a unix user while running task
+instances based on the task's ``run_as_user`` parameter, which takes a user's name.
+
+*NOTE* For impersonations to work, Airflow must be run with `sudo` as subtasks are run
```

Comment:
This is the section I was referring to that wouldn't work even with a sudoers file if I understand correctly (without giving the airflow user root access):
https://github.com/apache/incubator-airflow/pull/1934/files#diff-d675f89f6745fb6b4ee2192b97b2f45fR60

Happy to discuss offline too, let me know.